taxonomy-editor-frontend/src/pages/editentry/ListEntryChildren.jsx

```diff
@@ -13,13 +13,12 @@ import ISO6391 from 'iso-639-1';
 import { ENTER_KEYCODE } from "../../constants";
 
 
-const ListEntryChildren = ({url, setUpdateNodeChildren}) => {
+const ListEntryChildren = ({url, urlPrefix, setUpdateNodeChildren}) => {
     const [relations, setRelations] = useState(null);
     const [newChild, setNewChild] = useState(null);
     const [newLanguageCode, setNewLanguageCode] = useState(null);
-    const [open, setOpen] = useState(false); // Used for Dialog component
-    const [btnDisabled, setBtnDisabled] = useState(true) // For enabling or disabling Dialog button
-    const [isValidLC, setisValidLC] = useState(false); // Used for validating a new LC
+    const [openDialog, setOpenDialog] = useState(false); // Used for Dialog component
+    const [isValidLanguageCode, setIsValidLanguageCode] = useState(false); // Used for validating a new LC
```

Comment:
could this be replaced by a derived state? makes it easier to maintain it in sync with the language code

Reply:
I think `isValidLanguageCode` should be a state, since the Button component needs to be disabled/enabled based on it.

Reply:
yes, think will be better to have it as derived state as it will change every time the `newLanguageCode` changes. And is easier to maintain in sync

Reply:
Yep, I made the changes.